backend/transcript/views.py

```diff
@@ -109,6 +105,8 @@ def create_youtube_transcription(request):
     transcript = Transcript.objects.filter(video_id__exact=video_id).filter(
         language=lang
     )
+    # TODO: Check if this is the expected workflow? In this case, if only a machine
```

Comment:
@reviewers please check this workflow? I am not sure if this is the expected workflow since @prakharrathi25 worked on this.